x-pack/plugins/canvas/public/lib/breadcrumbs.ts

```diff
@@ -47,8 +47,6 @@ export const getBaseBreadcrumb = (history: History): ChromeBreadcrumb => {
   };
 };
 
-export const getWorkpadBreadcrumb = ({
-  name = 'Workpad',
-}: { name?: string } = {}): ChromeBreadcrumb => ({
+export const getWorkpadBreadcrumb = ({ name }: { name: string }): ChromeBreadcrumb => ({
```

Comment:
Can you clarify why you got rid of `getWorkpadBreadcrumb = ({name = 'Workpad} ...)`? I assume this code was meant to make it so that, if the title was set to empty, the breadcrumb would simply show "Workpad" instead of the long ID that it does now. Did this functionality not work, or were you just making behaviour consistent between the breadcrumb and the listing page?

Reply:
Oh I just thought it'd be more useful to have something less vague than "Workpad", but honestly, you're right that reading out a long id string is probably a worse UX. I can put it back to Workpad, but I was thinking maybe it should read like "Untitled workpad" so they're prompted to add a title. I'll remove all the logic where I set it to the workpad ID.

Reply:
I definitely agree that `Untitled workpad` would probably be better than simply `Workpad` - that's more consistent with how we do it in Dashboard as well, so that's a bonus :) But yeah, I think avoiding the ID and going with either `Workpad` like it was before or, ideally, `Untitled workpad` would be the best solution 👍